src/pages/packer/plugins/index.tsx

```diff
@@ -42,8 +42,11 @@ async function getStaticProps(ctx) {
 	 */
 	if ('props' in staticProps) {
 		// Partial nav data is provided from base getStaticProps, in menuItems
-		const partialNavData =
+		const baseMenuItems =
 			staticProps.props.layoutProps.sidebarNavDataLevels[2].menuItems
+		// Remove the authored title element and "Overview" link, these
+		// would be redundant with the styled overview item that we add.
+		const partialNavData = [baseMenuItems[0], ...baseMenuItems.slice(3)]
```

Comment:
🎏  Feels a little risky since this logic could break if the order of the menu items is edited, right? Not sure how else to go about it though 🤔 Have we considered removing those items in the source content instead? Or at least the 'overview'?

Reply:
Mostly speaking to the `slice` here and in the previous file.

Reply:
Have added a `addBrandedOverviewSidebarItem` helper to make the logic more robust - will add detail in the main comment thread shortly 👍